functional-tests/fn_bridge_deposit_seq_unreliable.py

```diff
@@ -0,0 +1,149 @@
+import time
+
+import flexitest
+from bitcoinlib.services.bitcoind import BitcoindClient
+from strata_utils import (
+    deposit_request_transaction,
+    get_balance,
+)
+from web3 import Web3
+from web3.middleware import SignAndSendRawMiddlewareBuilder
+
+import testenv
+from constants import (
+    DEFAULT_ROLLUP_PARAMS,
+    SATS_TO_WEI,
+)
+from utils import get_bridge_pubkey, wait_until, wait_until_with_value
+
+# Local constants
+# D BTC
+DEPOSIT_AMOUNT = DEFAULT_ROLLUP_PARAMS["deposit_amount"]
+# Gas for the withdrawal transaction
+WITHDRAWAL_GAS_FEE = 22_000  # technically is 21_000
+# Ethereum Private Key
+# NOTE: don't use this private key in production
+ETH_PRIVATE_KEY = "0x0000000000000000000000000000000000000000000000000000000000000001"
+# BTC Operator's fee for withdrawal
+OPERATOR_FEE = DEFAULT_ROLLUP_PARAMS["operator_fee"]
```

Comment:
Spaces between each of these after each assignment before the next comment to make them easier to read.